src/electrostatics.jl

```diff
@@ -0,0 +1,87 @@
+"""
+`module Electro`
+
+This module implements various functions related to (long range) electrostatic interactions
+"""
+
+module Electro
```

Comment:
would it be ok to rename it `Electrostatics`

Reply:
Of course, I am happy to rename it.